.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container: debian:bullseye
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          apt-get update && apt-get install -y --no-install-recommends make python3-pip
```

Nit (here and continuing below): Suggest expanding short flags in scripting for explicitness and readability:

```suggestion
          apt-get update && apt-get install --yes --no-install-recommends make python3-pip
```